### PR TITLE
Add new properties, new characteristics, Generic Property messages

### PR DESCRIPTION
**Pull request by wiktorzbroja-silvair**

### CODE REVIEW

**Review by bartekbartnicki-silvair — APPROVED** [review]
- checked with spec
- all GenericPropertyOpcode tested
- not all Properties tested by UT (IMO, we don't want check all of them, I don't see added value of UT which check all properties IDs - we should check it manually by comparing with specification)
- we have discuss with Wiktor about validating if values of properties are valid:
  - From our perspective (testers) in same cases we want to prepare invalid message
  - In other place in bluetooth_mesh we do not validate it. It could be introduce in future after decision that we want do it globally and after agree how we want to do it.
  - In FW tests we could want validate if FW do not send invalid values of properties. According "Explicit is better than implicit" I prefer to write tests where we explicit check value of property than to excepting exception from bluetooth_mesh.

**Comment by bartekbartnicki-silvair** [comment]
@michallowasrzechonek-silvair From my perspective ready to merge and publish

**Comment by kamiljanowski-silvair** [comment]
Please, add payloads that can be created to `test_capnproto.py` file

**Review by kamiljanowski-silvair — CHANGES_REQUESTED** [review]
add tests with example payloads (the best with different msg format which will be used in Silvair Platform) to test_capnproto.py

**Review by michallowasrzechonek-silvair — APPROVED** [review]
LGTM!

File: bluetooth_mesh/messages/generic/property.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+GenericPropertiesStatus = Struct(
+    "property_ids" / GreedyRange(SensorPropertyId),
+)
+
+GenericPropertyGet = Struct(
+    "property_id" / SensorPropertyId,
+)
+
+PropertyStatus = Struct(
```
Comment (kamiljanowski-silvair) [inline bluetooth_mesh/messages/generic/property.py:63]: This shall not be removed?

**Review by kamiljanowski-silvair — APPROVED** [review]

File: bluetooth_mesh/messages/generic/property.py (hunk 1 above)
Comment (wiktorzbroja-silvair) [reply bluetooth_mesh/messages/generic/property.py]: True

**Review by kamiljanowski-silvair — CHANGES_REQUESTED** [review]
Please, clean commit history. I prefer to have 3 commits:

1) Add Generic Property messages + tests
2) reformat other payloads in test_capnproto
3) Black + isort reformat

**Review by kamiljanowski-silvair — APPROVED** [review]
Good job :)

**Pull request merged**